parsons/google/google_cloud_storage.py

```diff
@@ -136,7 +136,14 @@ def delete_bucket(self, bucket_name, delete_blobs=False):
         bucket.delete(force=delete_blobs)
         logger.info(f"{bucket_name} bucket deleted.")
 
-    def list_blobs(self, bucket_name, max_results=None, prefix=None, match_glob=None):
+    def list_blobs(
+        self,
+        bucket_name,
+        max_results=None,
+        prefix=None,
+        match_glob=None,
+        include_file_details=False,
```

Comment:
Could you update the documentation of the return type to account for `include_file_details`?

Reply:
Added to the docstring - I linked to Google's reference docs here but can't remember if that's bad practice or not (happy to remove if so)